ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -370,33 +384,47 @@ const GENE_DETAIL_SECTIONS = [
     color: 'pink',
     description: 'RNA-Seq Outlier',
     label: 'RNA-Seq',
-    showDetails: (gene, rnaSeqData) => rnaSeqData && rnaSeqData[gene.geneId],
-    detailsDisplay: (gene, rnaSeqData) => (
-      <div>
-        This gene is flagged as an outlier for RNA-Seq in the following samples
```

Comment:
This can not be removed, it is not an acceptable change. Add it back

Reply:
Okay, I'll add the message back.